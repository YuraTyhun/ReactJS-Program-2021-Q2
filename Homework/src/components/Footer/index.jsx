import React from 'react';

import Logo from '../Logo';

import './Footer.scss';

const Footer = () => (
  <footer className="footer-container">
    <Logo />
  </footer>
);

export default Footer;
