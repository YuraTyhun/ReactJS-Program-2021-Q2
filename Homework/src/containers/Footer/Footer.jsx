import './Footer.scss';
import Logo from '../../components/Logo';

import React from 'react';

const Footer = () => (
    <footer className="footer-container">
        <Logo />
    </footer>
);

export default Footer;