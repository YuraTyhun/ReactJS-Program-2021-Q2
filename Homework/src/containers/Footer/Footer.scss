.footer-container {
    display: flex;
    justify-content: center;
    background-color: #424242;
    padding: 15px 0;
}