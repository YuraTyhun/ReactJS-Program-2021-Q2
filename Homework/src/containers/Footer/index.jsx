import React from 'react';

import Logo from '../../components/Logo';

import './Footer.scss';

const Footer = () => (
    <footer className="footer-container">
        <Logo />
    </footer>
);

export default Footer;