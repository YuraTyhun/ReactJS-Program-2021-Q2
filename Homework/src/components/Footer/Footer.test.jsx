import React from 'react';
import renderer from 'react-test-renderer';

import Footer from './index';

it('Footer renders correctly', () => {
    const footer = renderer
        .create(<Footer />)
        .toJSON();
    expect(footer).toMatchSnapshot();
})