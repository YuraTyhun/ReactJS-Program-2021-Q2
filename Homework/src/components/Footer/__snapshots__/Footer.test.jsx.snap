// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="footer-container"
>
  <div
    className="logo"
  >
    <span
      className="netflix"
    >
      netflix
    </span>
    <span
      className="roulette"
    >
      roulette
    </span>
  </div>
</footer>
`;
